copyright.md

```diff
@@ -17,10 +15,8 @@ The manual is provided “as is” without expressed or implied warranty. You ma
 *	https://computation.llnl.gov/casc/sundials/main.html
```

Comment:
Sundials links are changed and are now:
https://computation.llnl.gov/projects/sundials/license 
https://computation.llnl.gov/projects/sundials